### PR TITLE
[qos]: Fill COUNTERS_QUEUE_TYPE_MAP table.

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**
Create queue to queue type mapping. Fill COUNTERS_QUEUE_TYPE_MAP table.

### CODE REVIEW

**Comment by oleksandrivantsiv** [comment]
Depends on https://github.com/Azure/sonic-swss-common/pull/164

File: orchagent/portsorch.cpp [hunk 1]
```diff
@@ -1502,6 +1504,37 @@ void PortsOrch::initializeQueues(Port &port)
...
+                queueType = "SAI_QUEUE_TYPE_ALL";
+                break;
+            case SAI_QUEUE_TYPE_UNICAST:
+                queueType = "SAI_QUEUE_TYPE_UNICAST";
+                break;
+            case SAI_QUEUE_TYPE_MULTICAST:
+                queueType = "SAI_QUEUE_TYPE_MULTICAST";
+                break;
+            default:
+                throw runtime_error("Unsupported queue " + to_string(queueIndex) +
```
Comment (qiluo-msft) [inline orchagent/portsorch.cpp]: >runtime_error [](start = 22, length = 13)

We prefer short fixed string in exception string. You may use SWSS_LOG_ERROR before it.

**Comment by qiluo-msft** [comment]
>         SWSS_LOG_ERROR("Failed to get queue list for port %s rv:%d", port.m_alias.c_str(), status);

This function become so longer that not friendly to reader. Please add some comments for each blocks.

---
Refers to: orchagent/portsorch.cpp:1477 in afa3db1. [](commit_id = afa3db1398370e17bc5c52c546b48ad0dd47df91, deletion_comment = False)

**Comment by oleksandrivantsiv** [comment]
@qiluo-msft I've moved functionality for getting queue type into separated functions. Which simplifies **initializeQueues** function. However I'm not going to modify and comment code for which I'm not responsible.

File: orchagent/portsorch.cpp [hunk 2]
```diff
@@ -1502,6 +1537,16 @@ void PortsOrch::initializeQueues(Port &port)
                 to_string(queueIndex));
         queueIndexVector.push_back(queueIndexTuple);
 
+
+        string queueType;
```
Comment (qiluo-msft) [inline orchagent/portsorch.cpp]: >queueType [](start = 15, length = 9)

Move into if-scope

**Comment by lguohan** [comment]
retest this please

**Comment by oleksandrivantsiv** [comment]
@lguohan This commit depends on https://github.com/Azure/sonic-swss-common/commit/6d91ade570ca8e2e4f21c76312625d52352d8fb4. To pass build sonic-swss-common submodule should be updated. I'll update it and submit PR today.

**Comment by lguohan** [comment]
retest this please

**Pull request merged**